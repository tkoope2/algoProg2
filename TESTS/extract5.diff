11c11
< @     PATH: <>
---
> @     PATH: <v5 v6 v7 v8 v9 v10 v11 v12 v13 v14 v15 v16 v17 v18>
17c17
< @     PATH: <>
---
> @     PATH: <v5>
19c19
< @     PATH: <>
---
> @     PATH: <v5 v6 v7>
21c21
< @     PATH: <>
---
> @     PATH: <v5 v6 v7 v8 v9 v10 v11>
23c23
< @     PATH: <>
---
> @     PATH: <v5 v6 v7 v8 v9 v10 v11 v12 v13 v14 v15 v16 v17 v18 v19>
25c25
< @     PATH: <>
---
> @     PATH: <v5 v6>
27c27
< @     PATH: <>
---
> @     PATH: <v5 v6 v7 v8>
29c29
< @     PATH: <>
---
> @     PATH: <v5 v6 v7 v8 v9 v10 v11 v12>
31c31
< @     PATH: <>
---
> @     PATH: <v5 v6 v7 v8 v9 v10 v11 v12 v13 v14 v15 v16 v17 v18 v19 v20>
33c33
< @     PATH: <>
---
> @     PATH: <v5 v6 v7 v8 v9>
35c35
< @     PATH: <>
---
> @     PATH: <v5 v6 v7 v8 v9 v10 v11 v12 v13>
39c39
< @     PATH: <>
---
> @     PATH: <v5 v6 v7 v8 v9 v10 v11 v12 v13 v14 v15 v16 v17>
41c41
< @     PATH: <>
---
> @     PATH: <v5 v6 v7 v8 v9 v10>
43c43
< @     PATH: <>
---
> @     PATH: <v5 v6 v7 v8 v9 v10 v11 v12 v13 v14>
45c45
< @     PATH: <>
---
> @     PATH: <v5 v6 v7 v8 v9 v10 v11 v12 v13 v14 v15>
47c47
< @     PATH: <>
---
> @     PATH: <v5 v6 v7 v8 v9 v10 v11 v12 v13 v14 v15 v16>
