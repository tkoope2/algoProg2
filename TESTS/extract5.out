@1++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
@2
@3 COMMAND-LINE:
@4
@5    ./dfs chain20.g v5 
@6
@7++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
@ v2
@     PATH: <>
@ v18
@     PATH: <>
@ v3
@     PATH: <>
@ v4
@     PATH: <>
@ v5
@     PATH: <>
@ v7
@     PATH: <>
@ v11
@     PATH: <>
@ v19
@     PATH: <>
@ v6
@     PATH: <>
@ v8
@     PATH: <>
@ v12
@     PATH: <>
@ v20
@     PATH: <>
@ v9
@     PATH: <>
@ v13
@     PATH: <>
@ v1
@     PATH: <>
@ v17
@     PATH: <>
@ v10
@     PATH: <>
@ v14
@     PATH: <>
@ v15
@     PATH: <>
@ v16
@     PATH: <>
